README.md

```diff
@@ -58,11 +58,17 @@ you need a local web server.
 * Or alternatively, use the provided minimal server, that also does the job,
   and includes a minimal server-side synchronization mechanism.
 
-  You can launch it via
+You can launch it via
 
-```
-./_obuild/*/learnocaml-simple-server.byte
-```
+```./learnocaml-simple-server.byte```.
+
+An Docker container can be generated using this minimal server, via the provided
+script: ```sudo sh scripts scripts/build-docker-image.sh -repo-dir ../learn-ocaml-repository```.
```

Comment:
Is it really necessary to be root to build the image? I believed that being in the `docker` group was enough.

Reply:
@yurug @OCamlPro-Couderc indeed being just in the `docker` group would suffice, but this could be quite insecure, see e.g. https://fosterelli.co/privilege-escalation-via-docker.html and https://docs.docker.com/engine/security/security/#docker-daemon-attack-surface ; so requiring to be root (using `sudo` or so) is certainly worth it.

Reply:
@erikmd That's right, I added a link to the documentation in the README.